legal/security.md

```diff
@@ -72,6 +91,13 @@ These notifications don't include any sensitive information other than commit re
 
 We store these build notifications for debugging purposes, and for debugging purposes only.
 
+### What data do we store from Bitbucket?
+When you push code to Bitbucket for a repository that is set up to run on Travis, we get a push notification. The same is true for pull requests that are sent to us.
```

Comment:
What about user info?

Reply:
please add everything you think is necessary